shared/src/androidMain/kotlin/org/icerock/template/main.android.kt

```diff
@@ -2,4 +2,5 @@ import androidx.compose.runtime.Composable
 
 actual fun getPlatformName(): String = "Android"
 
-@Composable fun MainView() = App()
+@Composable
+fun MainView() = App()
```

Comment:
rollback please this change. and moving of file too

Reply:
done